pkg/crc/preflight/preflight_checks_linux.go

```diff
@@ -80,13 +80,9 @@ func checkKvmEnabled() error {
 
 func fixKvmEnabled() error {
 	logging.Debug("Trying to load kvm module")
-	cmd := exec.Command("modprobe", "kvm")
-	buf := new(bytes.Buffer)
-	cmd.Stderr = buf
-	err := cmd.Run()
+	stdOut, stdErr, err := crcos.RunWithDefaultLocale("modprobe", "kvm")
 	if err != nil {
-		logging.Debugf("%v : %s", err, buf.String())
-		return fmt.Errorf("Failed to load kvm module")
+		return fmt.Errorf("Failed to load kvm module: %s %v: %s", stdOut, err, stdErr)
```

Comment:
Fwiw, what would be nice would be to define an `RunError` type
```
type RunError struct {
    stdOut string
    stdErr string
    err Error
}
```
and make use of `return fmt.Errorf("Failed to load kvm module: %w", runError)
(see https://blog.golang.org/go1.13-errors )

Reply:
@cfergeau I think then this should go as part of runCmd where we can wrap `err` and `stderr` as part of struct.